Add WhiteSource config to exclude Docusaurus Dev Dependencies

mcleo-d opened the pull request:
### Description

The following WhiteSource config changes were made to the project ...

* `"configMode": "LOCAL"` added to `.whitesource`
* `whitesource.config` added to project `root`
* `**/website/node_modules/**` added to `whitesource.config`
* `package-lock.json` updated to latest version of node modules
* Updated ODP WhiteSource documentation URL

This PR resolves the following issues ...

* Resolves #1
* Resolves #2
* Resolves #3
* Resolves #4
* Resolves #5
* Resolves #6
* Resolves #7
* Resolves #8
* Resolves #11

maoo commented on whitesource.config:
```diff
@@ -0,0 +1,315 @@
+###############################################################
+# WhiteSource custom configuration file
+# To enable WS integration on a FINOS repo, email help@finos.org
+# Please read docs on https://finosfoundation.atlassian.net/wiki/spaces/FDX/pages/1129283585/WhiteSource+for+GitHub.com
```
We should link to https://odp.finos.org/docs/development-infrastructure/code-validation/whitesource instead. And also change the project blueprint accordingly of course.

mcleo-d replied:
This is done and committed 👍

maoo commented:
LGTM!

mcleo-d commented:
Hi @maoo - I have updated the `package-lock.json` to the latest version and updated the ODP documentation URL. It would be great if you could review, accept and merge as appropriate.
